packages/shared/forks/ReactFeatureFlags.test-renderer.js

```diff
@@ -48,7 +48,6 @@ export const enableFilterEmptyStringAttributesDOM = false;
 export const disableNativeComponentFrames = false;
 export const skipUnmountedBoundaries = false;
 export const deletedTreeCleanUpLevel = 3;
-export const enableSuspenseLayoutEffectSemantics = false;
```

Comment:
Might be worth testing an ad-hoc sync to confirm turning this on doesn't break tests